### PR TITLE
Add APIs to query ondemand Pod log

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/changelog/docs/en/changes/changes.md).

### CODE REVIEW

File: oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/GraphQLQueryConfig.java [hunk 1]
```diff
@@ -31,4 +31,5 @@ public class GraphQLQueryConfig extends ModuleConfig {
     private boolean enableLogTestTool;
     private int maxQueryComplexity = 1000;
     private boolean enableUpdateUITemplate = false;
+    private boolean enableOndemandLog = false;
```
Comment (wu-sheng) [inline oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/GraphQLQueryConfig.java:34]:
```suggestion
    private boolean enableOnDemandLog = false;
```
The most I saw about this is `on-demand` or `onDemand` as two words.

**Comment by kezhenxu94** [comment]
> image
> 
> We should add `on-demand logs` here.
> 
> And one question, on which dashboards we are going to add this feature? `Service Mesh` and `Kubernetes`?

I discussed with @Fine0830 and want to put it here 
Screen Shot 2022-05-25 at 21 30 38

**Comment by wu-sheng** [comment]
This is good, recommend considering putting into k8s dashboard as well.

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
Do we have any chance to have e2e for this? Considering the env limitations, is this practical?

**Pull request merged**